pkgs/development/python-modules/yubico/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, pytest, pyusb }:
+
+buildPythonPackage rec {
+  pname   = "python-yubico";
+  version = "1.3.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1gd3an1cdcq328nr1c9ijrsf32v0crv6dgq7knld8m9cadj517c7";
+  };
+
+  checkInputs = [ pytest ];
+  propagatedBuildInputs = [ pyusb ];
+
+  checkPhase =  ''
+    py.test -k "not usb"
+  '';
```

Comment:
Missing pythonImportsCheck

Reply:
Not sure how to implement this. Can you give me an example?

Reply:
https://github.com/NixOS/nixpkgs/blob/111ebb0f076cac006ce31dd87bfc3cdfa69b423d/pkgs/applications/misc/bikeshed/default.nix#L62